src/components/AddContent/helpers.ts

```diff
@@ -29,6 +30,20 @@ export const mapFormikToAPIValues = (
     gpgKey,
   }));
 
+const mapNoMetaDataError = (validationData: ValidationResponse) =>
```

Comment:
This function fixes the noMetaData errorstate that was discussed with design and @jlsherrill